lib/senkyoshi/models/content.rb

```diff
@@ -45,20 +53,23 @@ def iterate_xml(xml, pre_data)
       end
       @type = xml.xpath("/CONTENT/RENDERTYPE/@value").first.text
       @parent_id = pre_data[:parent_id]
-      bb_type = xml.xpath("/CONTENT/CONTENTHANDLER/@value").first.text
-      bb_type.slice! "resource/"
-      @module_type = CONTENT_TYPES[bb_type]
+      @module_type = self.class.name.gsub("Senkyoshi::", "")
```

Comment:
I was thinking about having something like this? 
```
    MODULE_TYPES = {
      "Senkyoshi::Attachment" => "Attachment",
      "Senkyoshi::Assignment" => "Assignment",
      "Senkyoshi::ExternalUrl" => "ExternalUrl",
      "Senkyoshi::WikiPage" => "WikiPage",
      "Senkyoshi::Quiz" => "Quizzes::Quiz",
    }.freeze
```
I added it to the code to try it out, but I haven't pushed it up. I can though if you think it is a way to go. I kind of like it because then you can also get rid of the 'Quizzes::Quiz' special case and treat all of the module item types the same way.